### PR TITLE
Update Newcomers_Guide.rst

### PR DESCRIPTION
**Pull request by HarshKhatore**
Newcomers_Guide.rst: documentation changes

Resolves the ambiguity between using Fixes or Closes for a PR

``Closes`` https://github.com/coala/documentation/issues/465

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by HarshKhatore** [comment]
@prakashdanish  How do I do that i.e. change the commit message?

**Comment by HarshKhatore** [comment]
@palash25  Okay. Regarding the issue URL, I have done exactly as you are telling. But, this issue is in another repo, which is being referred here( in coala/documentation ).

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -333,6 +333,11 @@ and clicking ``New Pull Request``.
     updated, you can edit your comment instead. If you need to add new
     information, make a new comment.
 
+    If you are generating a pull request for a particular issue, you need to
+    specify the issue number that your pull request resolves. If an issue is
+    labeled as a ``bug``, use ``Fixes XXXX`` and if it's not labeled as
+    ``bug``, ``use Closes XXXX``, where ``XXXX`` denotes the issue number.
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: -->
```
use ``Closes XXXX``
```

Comment (Makman2) [reply docs/Developers/Newcomers_Guide.rst]: But maybe you should just refer to "writing_good_commits"?

**Comment by Makman2** [comment]
Please also update `Writing_Good_Commits.rst`, I think this is the main place where we should have it, and trying to refer to it from the newcomers-guide :+1:

File: docs/Developers/Newcomers_Guide.rst (hunk 1 above)
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: 1. "denotes the issue number": Try to say "denotes the full issue URL" :+1:
2. Maybe change the format to `Fixes <full-issue-url>`/`Closes <full-issue-url>` or similar?

Comment (HarshKhatore) [reply docs/Developers/Newcomers_Guide.rst]: @Makman2 Okay, thanks for the review. I will try to correct the things.

Comment (Makman2) [reply docs/Developers/Newcomers_Guide.rst]: Sorry it took so long, sometimes PRs are missed :/ In that case feel free to ping after a while so we get new notifications ;)

Comment (HarshKhatore) [reply docs/Developers/Newcomers_Guide.rst]: @Makman2 I have done the required changes, please see and guide if any more corrections are required. Also, as you proposed to update Writing_Good_Commits.rst, so, should I open a new PR for that now?

**Comment by prnvdixit** [comment]
@HarshKhatore Please look at the issues like mixing up of tabs - spaces and length of a line (i.e. 83 chars) being larger than that allowed by convention (i.e. 80 chars). Running ```coala``` on the changed file might help you in updating them.

Also, the Shortlog expects the first letter after ```:``` to be a capital letter but in your commit it is ```d``` - please update that too in the commit message. :+1:

**Comment by Makman2** [comment]
@HarshKhatore still on it? :)

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -333,6 +333,12 @@ and clicking ``New Pull Request``.
     updated, you can edit your comment instead. If you need to add new
     information, make a new comment.
 
+    If you are generating a pull request for a particular issue, you need to
```
Comment (troublemagnet) [inline docs/Developers/Newcomers_Guide.rst]: I think generating is not the most appropriate word that can be used here. Perhaps you could write something like "If you are making.."

**Comment by HarshKhatore** [comment]
@Makman2 No, not for now.

**Comment by Makman2** [comment]
Alright closing then :)
I'll unassign you from the issue. If you want to take it up again, get assigned and reopen the PR :+1:

**Pull request closed without merging**